app/helpers/maybe_form_helper.rb

```diff
@@ -0,0 +1,7 @@
+module MaybeFormHelper
+  # Returns form_with that uses the MaybeFormBuilder
+  def maybe_form_with(**options, &block)
+    options[:builder] ||= MaybeFormBuilder
+    form_with(**options, &block)
+  end
```

Comment:
I think we could use `default_form_builder TailwindFormBuilder` in application_controller and then the default form builder will be that, and we don't need to remember to say `maybe_form_with` we can just use `form_with` as normal.   I think the goal should be to be able to use normal Rails conventions.